datafusion/datasource-parquet/src/opener.rs

```diff
@@ -55,8 +57,8 @@ pub(super) struct ParquetOpener {
     pub limit: Option<usize>,
     /// Optional predicate to apply during the scan
     pub predicate: Option<Arc<dyn PhysicalExpr>>,
-    /// Schema of the output table
-    pub table_schema: SchemaRef,
+    /// Schema of the file as far as the rest of the system is concerned.
+    pub logical_file_schema: SchemaRef,
```

Comment:
I think this rename is worth it - there's been constant confusion even amongst maintainers about this. And this is only public for internal use.